src/fragments/lib/auth/flutter/getting_started/10_preReq.mdx

```diff
@@ -1,8 +1,27 @@
-* A Flutter application targeting Flutter SDK >= 2.10.0 (stable version) with Amplify libraries integrated
+<BlockSwitcher>
+
+<Block name="Stable (Mobile)">
+
+* A Flutter application targeting Flutter SDK >= 2.10.0 with Amplify libraries integrated
   * An iOS configuration targeting at least iOS 11.0
   * An Android configuration targeting at least Android API level 21 (Android 5.0) or above
   * For a full example please follow the [project setup walkthrough](/lib/project-setup/create-application)
 
-import flutter0 from "/src/fragments/lib/project-setup/native_common/prereq/flutter_null_safety.mdx";
+</Block>
+
+<Block name = "Developer Preview (Mobile, Web & Desktop)">
+
+* A Flutter application targeting Flutter SDK >= 3.0.0 with Amplify libraries integrated
+  
+  The following are also required, depending on which platforms you are targeting:
+
+  * An iOS configuration targeting at least iOS 13.0 and XCode version >=13.2
+  * An Android configuration targeting at least Android API level 24 (Android 7.0) or above
+  * Any browser supported by Flutter web
+  * Any macOS or Windows OS meeting Flutter minimums. 
+  * Any Ubuntu Linux distribution meeting Flutter minimums.
```

Comment:
Why these two are separated?

Reply:
Just trying to keep the bullet points concise.